shell/platform/darwin/macos/framework/Source/FlutterCompositor.h

```diff
@@ -9,11 +9,12 @@
 #include <list>
 
 #include "flutter/fml/macros.h"
-#import "flutter/shell/platform/darwin/macos/framework/Source/FlutterPlatformViewController.h"
-#include "flutter/shell/platform/darwin/macos/framework/Source/FlutterView.h"
-#include "flutter/shell/platform/darwin/macos/framework/Source/FlutterViewProvider.h"
 #include "flutter/shell/platform/embedder/embedder.h"
 
+#import "flutter/shell/platform/darwin/macos/framework/Source/FlutterPlatformViewController.h"
+#import "flutter/shell/platform/darwin/macos/framework/Source/FlutterView.h"
+#import "flutter/shell/platform/darwin/macos/framework/Source/FlutterViewProvider.h"
+
```

Comment:
These guys (unfortunately) are meant to be interleaved with includes in a single group according to the style guide

Reply:
Looks a bit weird, but sure :) Fixed.